testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/AbstractAdvancedBrokerTest.java

```diff
@@ -190,7 +189,7 @@ public void testRetrieveToken() throws Exception {
         logInWithBroker(bc);
         updatePasswordPage.updatePasswords("password", "password");
         waitForAccountManagementTitle();
-        accountUpdateProfilePage.assertCurrent();
+        Assert.assertFalse(loginPage.isCurrent());
```

Comment:
Should we updated to login to test app, and check that is opened rather the checking login page isn't current page